### DISCUSSION TITLE
Different Zoom Levels on one Map

### DISCUSSION THREAD

**Discussion by Arriba999** [systemed/tilemaker#647] [General]
Hi!

Is it possible to render tiles with different zoom levels one one map? I have a big map from germany with a bit netherlands an great britain, so it fits a widescreen container on a homepage when it's zoomed out.
The customer has 4 stores in 4 different citys. It would be interesting when the surroundings of the stores a rendered with zoom level 14, so that the customer can zoom in and look where the store is. But the rest of the map isn't needed to have the possibility that the user can zoom in so close.

If i render the whole map with level 14 i get more than 130.000 tiles.

Kind regards, Sascha

**Comment by systemed** [comment]
Yes, it is. You can run tilemaker several times for all the areas/zoom levels you want to generate. Write to the same .mbtiles, and use the `--merge` switch from the second time onwards. This will add to the existing file, rather than overwriting it.

**Comment by Arriba99** [comment]
Thank you!!
